BaseFilter/sections/specific.txt

```diff
@@ -756,6 +756,7 @@ pornhex.com##div[id^="fallbackSidebar"]
 pornhex.com#?#.related-items > div > p.text-xs:contains(ads)
 pornhex.com#%#//scriptlet('prevent-addEventListener', 'load', 'adIframe')
 besthealthmag.ca##.pre-article-ad
+nsmb.com##.Vt8kMlfQ6G73bU
```

Comment:
Anti-adblock notice is inside ad placeholder so in my opinion is a correct place, but I would prefer something more specific for ads.
I guess this rule (I mean `nsmb.com##.Vt8kMlfQ6G73bU`) could be moved to `! SECTION: Anti-adblock - Regular rules` in https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/Popups/sections/antiadblock.txt and additionally we could add (also to AnnoyancesFilter):
```adblock
nsmb.com#%#//scriptlet('prevent-xhr', 'pagead2.googlesyndication.com')
```

And in Base filter we could add rules to hide ad placeholders like:
```adblock
nsmb.com##.block_a_d
nsmb.com###header-banner
```